WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Notes Section/Customer Note/EditCustomerNoteViewModel.swift

```diff
@@ -84,14 +92,104 @@ final class EditCustomerNoteViewModel: EditCustomerNoteViewModelProtocol {
 private extension EditCustomerNoteViewModel {
     /// Calculates what navigation trailing item should be shown depending on our internal state.
     ///
-    private func bindNavigationTrailingItemPublisher() {
+    func bindNavigationTrailingItemPublisher() {
         Publishers.CombineLatest($newNote, performingNetworkRequest)
-            .map { [order] newNote, performingNetworkRequest -> EditCustomerNoteNavigationItem in
-                guard !performingNetworkRequest else {
```

Comment:
This logic becomes a bit more complex because we also have to check the value of the feature flag.